src/ci/run.sh

```diff
@@ -126,13 +126,13 @@ trap datecheck EXIT
 SCCACHE_IDLE_TIMEOUT=10800 sccache --start-server || true
 
 if [ "$RUN_CHECK_WITH_PARALLEL_QUERIES" != "" ]; then
-  $SRC/configure --enable-parallel-compiler
+  "$SRC"/configure --enable-parallel-compiler
   CARGO_INCREMENTAL=0 $PYTHON ../x.py check
   rm -f config.toml
   rm -rf build
 fi
 
-$SRC/configure $RUST_CONFIGURE_ARGS
+"$SRC"/configure "$RUST_CONFIGURE_ARGS"
```

Comment:
This doesn't look correct. This used to be many different arguments, like `--set x --set y`; now it's one argument all at once. I think you should change it back to how it was before. If you want to silence the shellcheck warning you could trying using an array instead but I'd revert it for now just to make sure it works.